src/core/server.cc

```diff
@@ -489,9 +489,10 @@ InferenceServer::InferAsync(std::unique_ptr<InferenceRequest>& request)
   }
 
 #ifdef TRITON_ENABLE_STATS
+  request->CaptureRequestStartNs();
   INFER_TRACE_ACTIVITY(
       request->Trace(), TRITONSERVER_TRACE_REQUEST_START,
-      request->CaptureRequestStartNs());
+      request->RequestStartNs());
 #endif  // TRITON_ENABLE_STATS
```

Comment:
EnsembleScheduler::Enqueue has the same problem of capturing a timestamp inside of a macro. Can you fix that as well (SequenceBatchScheduler::Enqueue has the correct code).